### PR TITLE
Fixed propagation event for flyout on decoders

### PR DESCRIPTION
**Pull request by yenienserrano**
### Description
Fixed flyout showing information related to a decoder showing information about another decoder when clicking on the button/link to open the content of the file containing the decoder.
 
### Issues Resolved
- #5219 

### Evidence

image

### Test
1. Navigate to Management/Decoders
2. Click on any row of the decoders table. A flyout will be opened with data from the selected decoder.
3. In the table of Related decoders clicks on a button/link of File column.
4. Check that the flyout data is not changed before closing

### Check List
- [x] All tests pass
  - [x] `yarn test:jest`
- [ ] New functionality includes testing.
- [ ] New functionality has been documented.
- [ ] Update [CHANGELOG.md](./../CHANGELOG.md)
- [x] Commits are signed per the DCO using --signoff

### CODE REVIEW

**Review by jbiset — APPROVED** [review]
# Wazuh dashboard

Legend:
:black_circle:: none
:green_circle:: pass
:yellow_circle:: warning
:red_circle:: fail
:white_circle:: not applicable

## UI

| Test | Chrome | Firefox | Safari |
| --- |  --- | --- | --- |
| Click on the file column of the decoders details table doesn't display data about the selected decoder and open the decoder content | :green_circle: | :green_circle: | :black_circle: |

**Details**

<details>

## Evidence

### Chrome - :green_circle:

Evidence_1

https://github.com/wazuh/wazuh-kibana-app/assets/43619595/23444712-52d6-4b43-b41a-5d10ab94672a

### Firefox - :green_circle:

Firefox_Evidence_1

Firefox_Evidence_2

</details>

**Comment by Desvelao** [comment]
# Wazuh dashboard

Legend:
:black_circle:: none
:green_circle:: pass
:yellow_circle:: warning
:red_circle:: fail
:white_circle:: not applicable

## UI

| Test | Chrome | Firefox | Safari |
| --- |  --- | --- | --- |
| Click on the file column of the decoders details table doesn't display data about the selected decoder and open the decoder content | :green_circle: | :black_circle: | :black_circle: |

**Details**
<details>
<summary>:green_circle: Click on the file column of the decoders details table doesn't display data about the selected decoder and open the decoder content</summary>

Chrome - :green_circle:
image
image

Firefox - :black_circle:

Safari - :black_circle:

</details>

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,12 @@
 
 All notable changes to the Wazuh app project will be documented in this file.
 
+## Wazuh v4.7.0 - OpenSearch Dashboards 2.7.0 - Revision 00
```
Comment (Desvelao) [inline CHANGELOG.md:5]: **nitpick**: Changelog is pointing to 4.7.0 but the base branch of the pull request is 4.6.0. What version do we want to add?

Comment (yenienserrano) [reply CHANGELOG.md:5]: Yes, it was pointing to 4.6.0 because the 4.7.0 branch was not yet created, I have already changed the branch.

**Review by Desvelao — DISMISSED** [review]
LGTM!

**Review by jbiset — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,14 +2,15 @@
 
 All notable changes to the Wazuh app project will be documented in this file.
 
-## Wazuh v4.7.0 - OpenSearch Dashboards 2.6.0 - Revision 01
+## Wazuh v4.7.0 - OpenSearch Dashboards 2.6.0 - Revision 00
```
Comment (Desvelao) [inline CHANGELOG.md:5]: **nitpick**: change to revision 01

Comment (Desvelao) [reply CHANGELOG.md]: Solved in https://github.com/wazuh/wazuh-kibana-app/pull/5597/commits/8f2d5bf48f9dbeff9664a5744a455b43071f5116.

**Review by Desvelao — APPROVED** [review]
LGTM!

**Pull request merged**